scripts/utils/package.js

```diff
@@ -118,7 +118,7 @@ function runInPackage({
  */
 function getAllPackagePaths() {
   const fullPaths = [];
-  const packagesDirs = ['packages/node_modules/@ciscospark', 'packages/node_modules/@webex'];
+  const packagesDirs = [ 'packages/@webex'];
```

Comment:
`packages/node_modules/@ciscospark` doenst have any package.json or src files. So when we run any command that uses this method it just fills the logs with ` Unable to find a package at <FILE_PATH>.